schema_salad/python_codegen.py

```diff
@@ -39,6 +39,14 @@
         "None_type", "_PrimitiveLoader(type(None))"
     ),
     "https://w3id.org/cwl/salad#Any": TypeDef("Any_type", "_AnyLoader()"),
+    "string": TypeDef("strtype", "_PrimitiveLoader((str, str))"),
```

Comment:
while you're at it, can you see what happens if you replace `(str, str)` with just `str` I think this is a legacy of the Python 2 -> Python 3 migration and exists because it used to be `(str, unicode)` but was updated via a simpleminded search-and-replace.